src/libraries/Common/src/Interop/OSX/Interop.Libraries.cs

```diff
@@ -5,17 +5,17 @@ internal static partial class Interop
 {
     internal static partial class Libraries
     {
-        internal const string CoreFoundationLibrary = "/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation";
-        internal const string CoreServicesLibrary = "/System/Library/Frameworks/CoreServices.framework/CoreServices";
-        internal const string CFNetworkLibrary = "/System/Library/Frameworks/CFNetwork.framework/CFNetwork";
+        internal const string CoreFoundationLibrary = "/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation.dylib";
+        internal const string CoreServicesLibrary = "/System/Library/Frameworks/CoreServices.framework/CoreServices.dylib";
+        internal const string CFNetworkLibrary = "/System/Library/Frameworks/CFNetwork.framework/CFNetwork.dylib";
```

Comment:
this is wrong, Frameworks don't have `.dylib` extensions. you'll need to revert these.

Reply:
Thanks!